.gitignore

```diff
@@ -1,4 +1,5 @@
 /.vscode
 /server/registration
 /server/templates/helpers/register.html
-/server/templates/helpers/_formHelpers.html
\ No newline at end of file
+/server/templates/helpers/_formHelpers.html
+/venv
```

Comment:
?

Reply:
You don't have to merge portion if you don't want to. I just made it so I wouldn't upload my virtual environment. I don't think I added the formHelpers line.